### DISCUSSION TITLE
What are the release triplets?

### DISCUSSION THREAD

**Discussion by AdelQraft** [microsoft/vcpkg#39979] [Q&A] [answered]
What exactly are the release triplets (eg. `x64-windows-release`, `x64-windows-static-md-release`, `x64-windows-static-release`? I can't believe there is no docs whatsoever anywhere… I guess they are the same as the normal triplets but more optimized, even though the normal triplets still work in release mode. Am I right?

**Comment by dg0yt** [comment, marked as answer]
Release triplets build only the release build type. 
Regular triplets build both debug and release configurations.
